src/components/List/List.stories.tsx

```diff
@@ -9,9 +9,9 @@ export default {
 } as Meta;
 
 const Template = args => {
-  const [selectedItem, setSelectedItem] = useState(args.selected);
+  const [selectedItem, setSelectedItem] = useState(args.selected || []);
```

Comment:
I'm not sure if you should change this since you're using the other template and not this one. This will affect all the list items